assets/javascripts/discourse/initializers/d-templates.js

```diff
@@ -42,47 +37,26 @@ function addOptionsMenuItem(api) {
     return {
       id: "discourse_templates_button",
       icon: "far-clipboard",
-      action: "showTemplatesButton",
+      action: "insertTemplate",
       label: "templates.insert_template",
     };
   });
 }
 
-function addKeyboardShortcut(api, container) {
+function addKeyboardShortcut(api) {
   const isMac = /Mac|iPod|iPhone|iPad/.test(navigator.platform);
   const modKey = isMac ? "meta" : "ctrl";
 
   api.addKeyboardShortcut(
     `${modKey}+shift+i`,
     (event) => {
       event.preventDefault();
+      const dTemplates = getOwner(this).lookup("service:d-templates");
```

Comment:
`this` here is the `Window`, not an owner object. Passing in the `container` might still be the best way to go.

Reply:
Done